src/dataservice/data.go

```diff
@@ -0,0 +1,330 @@
+
+package main
+
+import (
+        "bytes"
+        "encoding/json"
+        "flag"
+        "fmt"
+        "io/ioutil"
+        "math/rand"
+        "net/http"
+        "net/url"
+        "strconv"
+        "time"
+
+        "github.com/vmware/harbor/src/common/models"
+        "github.com/manveru/faker"
+)
+
+// Harbor deployment URL
+var baseURL string
+
+// This instantiates a Faker module used to generate randomized data
```

Comment:
Does Faker generate random data deterministically or can it be made to do so? Having that behavior would likely make the validations we have to do in the tests employing this tool easier to write.